bbb-libreoffice/assets/convert_pptx_with_bullet_and_autofit_fixes.py

```
@@ -0,0 +1,1183 @@
+def normalize_slide_xml(
+    xml_bytes: bytes,
+    *,
+    part_name: str,
+    stats: BulletNormalizationStats,
+    verbose: bool,
+) -> tuple[bytes, bool]:
+    """Normalize bullet indentation in one slide XML part."""
+    register_source_namespaces(xml_bytes)
+
+    try:
+        root = ET.fromstring(xml_bytes)
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)